test/second_q/operators/test_fermionic_op.py

```diff
@@ -84,7 +84,7 @@ def test_compose(self):
             ) @ FermionicOp({"": 1, "-_0 +_1": 1}, num_spin_orbitals=2)
             fer_op = fer_op.simplify()
             targ = FermionicOp(
-                {"+_0 +_1 -_1": 1, "-_0 +_0 -_1": 1, "+_0 -_0 +_1": 1, "-_0 -_1 +_1": -1},
+                {"+_0 +_1 -_1": 1, "-_0 +_0 -_1": 1, "+_0 +_1 -_0": 1, "-_0 -_1 +_1": -1},
```

Comment:
You swapped the order of the terms here, but you didn't add a minus sign.

Reply:
Thanks for spotting this! https://github.com/Qiskit/qiskit-nature/pull/900 should address this bug :+1: